server/index.js

```diff
@@ -41,17 +43,33 @@ app.post('/reservation', function (req, res) {
   var booking = req.body;
 
   Reservation.make(booking)
```

Comment:
I would assume that a reservtion_id is generated on the client side and included in the "booking" passed into the function. This would be the easiest way as the schema includes this category.